corehq/apps/app_manager/util.py

```diff
@@ -409,46 +409,83 @@ def _add_ancestors(ctype, generation):
     else:
         generations = []
 
+    subsets = [{
+        "id": generation_names[i],
+        "name": "{} ({})".format(generation_names[i], " or ".join(ctypes)) if i > 0 else base_case_type,
+        "key": "@case_type",
```

Comment:
It's supposed to be the property that the `id` references. However, I see that's not true anymore because of the way `parent` and `grandparent` case references are now constructed. It could theoretically be removed since it's not currently used by Vellum for these (`#case`) subsets, although it's not hurting anything either.